confignode/src/main/java/org/apache/iotdb/confignode/manager/ConfigManager.java

```diff
@@ -92,6 +95,8 @@ public class ConfigManager implements Manager {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(ConfigManager.class);
 
+  private volatile boolean isStopped = false;
```

Comment:
> Because the node status are not only run and stop, maybe you can use _org.apache.iotdb.commons.cluster.NodeStatus_ to maintain?
![image](https://user-images.githubusercontent.com/48384887/174600321-b5705e23-7610-40fa-ad09-98e6bdd69069.png)
already revert.